fala/templates/adviser/search.html

```diff
@@ -18,6 +18,20 @@ <h1 class="govuk-heading-xl"> Find a legal aid adviser or family mediator </h1>
   {% block search_form %}
       <div class="govuk-grid-column-full">
         <div class="laa-fala__grey-box">
+          {% if (form.postcode and form.postcode.errors) or (form.name and form.name.errors) %}
+            <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+              <h2 class="govuk-error-summary__title" id="error-summary-title">
+                There is a problem
+              </h2>
+              <div class="govuk-error-summary__body">
+                <ul class="govuk-list govuk-error-summary__list">
+                  {{ Form.render_field_errors_for_summary(form.postcode) }}
+                  {{ Form.render_field_errors_for_summary(form.name) }}
+                </ul>
+              </div>
+            </div>
+          {% endif %}
+
```

Comment:
Good spot!

Reply:
This is inside the grey box, but I think it'll be better outside of it. We'll need to check with someone to approve that as there is no design for us in figma for the errors :(